main.go

```diff
@@ -1,38 +1,47 @@
 package main
 
 import (
-	_ "embed"
 	"flag"
 	"log"
 	"runtime/debug"
 )
 
-//go:embed VERSION
-var version string
+var (
+	tag = "?"
```

Comment:
We should use a better default here. ```gerberos ?``` on stdout ight be confusing, ```gerberos unknown version``` might  be more understandable.

Reply:
(for example, I always use ```go build``` instead of ```make``` and others might do so too)

Reply:
Revised.